src/Resolver/WishlistCookieTokenResolver.php

```diff
@@ -32,10 +32,15 @@ public function resolve(): string
     {
         $wishlistCookieToken = $this->requestStack->getMainRequest()->cookies->get($this->wishlistCookieToken);
 
-        if (!$wishlistCookieToken) {
-            return (string) new WishlistToken();
+        if ($wishlistCookieToken !== null) {
+            return $wishlistCookieToken;
         }
 
-        return $wishlistCookieToken;
+        $wishlistCookieToken = $this->requestStack->getMainRequest()->attributes->get($this->wishlistCookieToken);
+        if ($wishlistCookieToken !== null) {
```

Comment:
Wouldn't it be better to use yoda style here, like above?

Reply:
yup